pkg/apis/sources/v1beta2/ping_types.go

```diff
@@ -81,6 +81,11 @@ type PingSourceSpec struct {
 	// Mutually exclusive with Data.
 	// +optional
 	DataBase64 string `json:"dataBase64,omitempty"`
+
+	// Date is the string of specific date time to fire events only once. Default is empty.
+	// Mutually exclusive with Schedule.
+	// +optional
+	Date string `json:"date,omitempty"`
```

Comment:
I don't think we should be modifying the older version.

Reply:
I think the CI doesn't allow the inconsistency of newer and older version because not modifying the older version cause CI fail

Reply:
we should delete v1beta2, but not in the PR. Basically this is fine.